### PR TITLE
Add addresses for de_CH

### PR DESCRIPTION
**Pull request by andre-dasilva**
### What does this changes

Add address formats for de_CH

### CODE REVIEW

**Comment by fcurella** [comment]
Thank you! ✨

File: faker/providers/address/de_CH/__init__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+               ('VD', 'Waadt'), ('VS', 'Wallis'), ('ZG', 'Zug'), ('ZH', 'Zürich'))
+
+    def canton(self):
+        """
+        Randomly returns a swiss canton ('Abbreviated', 'Name').
+        :example ('ZH', 'Zürich')
+        """
+        return self.random_element(self.cantons)
+
+    def administrative_unit(self):
```
Comment (m-aciek) [inline faker/providers/address/de_CH/__init__.py:30]: Can we add an alias from state() to administrative_unit()? Not adding the state breaks backwards compatibility.

@fcurella @andre-dasilva

Comment (m-aciek) [reply faker/providers/address/de_CH/__init__.py:30]:
```
  File "/usr/local/lib/python3.9/site-packages/faker/proxy.py", line 105, in __getattr__
    return getattr(self._factories[0], attr)
AttributeError: 'Generator' object has no attribute 'state'
```

Comment (andre-dasilva) [reply faker/providers/address/de_CH/__init__.py:30]: @m-aciek thats good for me.

**Pull request merged**